docs/impact/index.md

```diff
@@ -1,48 +1,17 @@
-# Impact
+# BIDS Impact
 
 This document intends to convey how the BIDS Steering and Maintainers Group can
 support your proposed BIDS grant. In an effort to express the impact of BIDS in
 the brain imaging community, we have shared several of our traffic metrics.
```

Comment:
```suggestion
BIDS document  intends to convey how the BIDS Steering and Maintainers Group can
support your proposed BIDS grant. In an effort to express the impact of BIDS in
the brain imaging community, we have shared several of our traffic metrics.
```
The Brain Imaging Data Structure (BIDS) is an open global community driving the standardization of neuroscience data across a broad and growing range of modalities and health research disciplines. First released in June 2016, it is supported by a worldwide research network and endorsed by organizations like the International Neuroinformatics Coordinating Facility ([INCF](https://www.incf.org/)). 
BIDS encompasses: 
* Over 40 domain-specific and modality-specific technical specifications across scientific and technical domains
* Open software conversion and analytics tools, and global infrastructure for collaborating on emerging standards in neuroscience ([Poldrack, et al, 2024](https://doi.org/10.1162/imag_a_00103))
* ~30,000 annual website visits from a very large community of neuroscience researchers actively consulting the BIDS specifications (April 2023-24). 
* Over 300 BIDS contributors currently supporting and maintaining BIDS community resources and tools
* Open working processes through online collaborative tools including GitHub, GitHub Pages, ReadTheDocs, Google Docs, and OSF 

The BIDS core steering and maintainers groups work with the community to optimize open adherence to FAIR principles (Findable, Accessible, Interoperable, and Reusable, [Wilkinson, et al, 2016](https://www.nature.com/articles/sdata201618)) and to actively lower barriers to adoption.

Reply:
note there are `read more` placeholder links i started adding, should be completed